segment-virtualenv.go

```diff
@@ -3,21 +3,28 @@ package main
 import (
 	"os"
 	"path"
+	"strings"
 
 	"gopkg.in/ini.v1"
 
 	pwl "github.com/justjanne/powerline-go/powerline"
 )
 
 func segmentVirtualEnv(p *powerline) []pwl.Segment {
-	var env string
+	env := os.Getenv("VIRTUAL_ENV_PROMPT")
+	if strings.HasPrefix(env, "(") && strings.HasSuffix(env, ") ") {
+		env = env[1 : len(env)-2]
```

Comment:
I think I'd prefer

```suggestion
		env = strings.TrimPrefix(env, "(")
		env = strings.TrimSuffix(env, ") ")
```

for clarity?

Reply:
Sure, done.